pkg/opentelemetry/output.go

```diff
@@ -181,9 +181,15 @@ func (o *Output) dispatch(entry metrics.Sample) error {
 		}
 
 		trend.Record(ctx, entry.Value, otelMetric.WithAttributes(MapTagSet(entry.Tags)...))
+	case metrics.Rate:
+		rate, err := o.getOrCreateCounter(name)
+		if err != nil {
+			return err
+		}
+
+		rate.Add(ctx, entry.Value, otelMetric.WithAttributes(MapTagSet(entry.Tags)...))
```

Comment:
This should be *two* counters as you need both the number of occurances and the actual number of possitive *or* negative values.

I don't know what is the preffered way this likely is good thing to ask around in the community or internally. 

Given that this will possibly evolve in the recommended output for k6 it will be nice for it to not have the problems due to which we remove stattsd.